src/dtm.jl

```diff
@@ -16,7 +16,17 @@ end
 #
 ##############################################################################
 
-# create col index lookup dictionary from a (sorted?) vector of terms
+"""
+```
+columnindices(terms)
```

Comment:
I guess this should be like the following - 
    columnindices(terms)

(Without backticks but has 4 spaced indent)
Refer: Point 1 in https://docs.julialang.org/en/v1/manual/documentation/index.html